src/main/java/com/google/codeu/data/Marker.java

```diff
@@ -0,0 +1,26 @@
+package com.google.codeu.data;
+
+public class Marker {
+
+  private double lat;
+  private double lng;
+  private String content;
+
+  public Marker(double lat, double lng, String content) {
+    this.lat = lat;
+    this.lng = lng;
+    this.content = content;
+  }
+
+  public double getLat() {
```

Comment:
Method descriptions. What do these return? Especially if these attributes are not described in the class description.

Reply:
Added